static/app/components/events/eventTagsAndScreenshot/dataSection.tsx

```diff
@@ -27,7 +27,7 @@ function DataSection({title, description, children, className, ...props}: Props)
           <GuideAnchor target={type} position="bottom">
             <Title>{title}</Title>
           </GuideAnchor>
-          <QuestionTooltip size="xs" position="top" title={description} />
+          <QuestionTooltip size="sm" position="top" title={description} />
```

Comment:
We should go through the app and make all xs icons -> sm for consistency

Reply:
yeah, and make them `gray300` instead of `gray200`, cause `gray200` is too light and not accessible

Reply:
Yah, let's do that after this PR